File: output/ChatGPT/avatar/Python/C++/codeforces_37_A.cpp
#include <iostream>
#include <vector>
#include <algorithm>
using namespace std;

int main() {
    int n;
    cin >> n;
    vector<int> l(n);
    for (int i = 0; i < n; i++) {
        cin >> l[i];
    }
    sort(l.begin(), l.end());
    int high = 1;
    int tower = n;
    int maximum = 1;
    for (int i = 1; i < n; i++) {
        if (l[i] == l[i - 1]) {
            tower--;
            high++;
        } else {
            if (high > maximum) {
                maximum = high;
            }
            high = 1;
        }
    }
    if (high > maximum) {
        maximum = high;
    }
    cout << maximum << " " << tower << endl;

    return 0;
}

File: output/CodeGen/avatar/Python/Java_handraft/codeforces_37_A.java
import java.util.*;

public class codeforces_37_A {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        System.out.println(maxTower(n, scanner));
    }
    public static String maxTower(int n, Scanner scanner) {
        int[] l = new int[n];
        for (int i = 0; i < n; i++) {
            l[i] = Integer.parseInt(scanner.next());
        }
        Arrays.sort(l);
        int tower = n;
        int high = 1;
        int maximum = 1;
        for (int i = 1; i < n; i++) {
            if (l[i] == l[i - 1]) {
                tower--;
                high += 1;
            } else {
                if (high > maximum) {
                    maximum = high;
                }
                high = 1;
            }
        }
        if (high > maximum) {
            maximum = high;
        }
        return maximum + " " + tower;
    }
}